osutils.pm

```diff
@@ -21,13 +21,19 @@ use warnings;
 use Carp;
 use base 'Exporter';
 use Mojo::File 'path';
+use bmwqemu 'diag';
+use POSIX ':sys_wait_h';
+use Symbol 'gensym';
+use IPC::Open3;
```

Comment:
Do we have that package in our distris? Couldn't find it. Is it included in perl?

Reply:
@okurz Yes, it is part of the core perl distribution

Reply:
Take your monday morning coffee, take a breath and then, luke, use the [force](http://www.letmegooglethat.com/?q=Perl+core+module+IPC%3A%3AOpen3)